packages/flutter/test/material/theme_data_test.dart

```diff
@@ -195,31 +195,31 @@ void main() {
     expect(theme.colorScheme.primary, const Color(0xff36618e));
     expect(theme.colorScheme.onPrimary, const Color(0xffffffff));
     expect(theme.colorScheme.primaryContainer, const Color(0xffd1e4ff));
-    expect(theme.colorScheme.onPrimaryContainer, const Color(0xff001d36));
```

Comment:
@Piinks – guessing you care about these types of changes...

Reply:
Our users do too. :)

These are the types of changes that are really annoying to users. In the past, updates like these require creating a migration guide so they can understand what changed, why and how to fix their apps.

Here is an example: https://docs.flutter.dev/release/breaking-changes/material-design-3-token-update